haskell_101/slides/haskell_101.tex

```diff
@@ -85,9 +97,9 @@ \section{Intro}
   \begin{itemize}
   \item Project environment
     \begin{itemize}
-    \item Cabal?
-    \item Stackage? Stack?
-    \item Haskell in Google3?
+    \item Cabal? Stack?
+    \item Hackage? Stackage?
+    \item Haskell at Google?
```

Comment:
Ideally, this item should be in the Google specific branch.
Not sure how much work is it.

Reply:
I think this should be ok, it's just a list of items we won't cover on the 101.